client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/backup-storage.tsx

```diff
@@ -67,7 +67,8 @@ const BackupStorageContent = ( {
 
 	// Show plugin name only if it is a activity from a plugin
 	const pluginName =
-		backup?.activityName.startsWith( 'plugin__' ) && backup.activityDescription[ 0 ]?.children[ 0 ];
+		backup?.activityName.startsWith( 'plugin__' ) &&
+		backup.activityDescription?.[ 0 ]?.children?.[ 0 ];
```

Comment:
I think this would always be an boolean value, but a few lines below we have the following:

```
{ pluginName ? ` - ${ pluginName }` : '' }
```

I'm not entirely sure what's this is supposed to render, but I might be reading this wrong?

Reply:
The `backup.activityDescription[0].children[0]` is a string value, not a boolean.

Reply:
Oh, I see! Thanks.